### ISSUE TITLE
Google Device Access "Something Went Wrong"

### ISSUE THREAD

**Issue by ZombieRoxtar** [koush/scrypted#1163]
When I select Login from the Google Device Access plugin, it always leads to "Something Went Wrong". This only started happening in October of 2023. I even tried removing the plugin and creating a brand new project, but it kept on happening.
Docker, 0.57.0
![image](https://github.com/koush/scrypted/assets/10099033/f9fcac36-9715-489a-8552-fbfbf6a190fc)

**Comment by MooreOfNick** [comment]
Yup, I am walking through the set up for these cameras and am running into the same issue.
